client/src/legacy/GridField.js

```diff
@@ -180,6 +194,36 @@ $.entwine('ss', function($) {
       if (hasLegacyFilterHeader) {
         this.find('.sortable-header th:last').html(content);
       }
+    },
+
+    keepStateInHistory: function() {
+      const newURLScheme = $(this).find('.gridfield-actionmenu__container').data('schema');
```

Comment:
```suggestion
      const newURLSchema = $(this).find('.gridfield-actionmenu__container').data('schema');
```
[URL Scheme](https://en.wikipedia.org/wiki/List_of_URI_schemes) means something specific - we're dealing with schema here.

Reply:
`newSchema` ?